.circleci/config.yml

```diff
@@ -270,8 +271,6 @@ workflows:
               only:
                 - /^release-.*/
           requires:
-            - lint
-            - unit_tests
```

Comment:
Removing these steps could release code that cannot start as an image, linting should only fail for serious issues (including TS related ones).

Reply:
The idea was to shorten the pipeline time by letting the push_to_gcr step kick off sooner. However, since step this no longer happens on every `develop` commit and only upon a `/^release-.*/`-tagged branch, you're right that the time gains don't justify the added risk. I've reverted the dependency change.